wg-data-protection/annual-report-2021.md

```diff
@@ -0,0 +1,45 @@
+# Data Protection WG Community Report 2021
+
+This report reflects back on CY 2020 and was written in Mar. 2021.
+
+## You and your role:
+
+When did you become a chair and do you enjoy the role?
+* Both Xiangqian Yu and Xing Yang became Data Protection WG co-chairs when the WG was formally established on Jan. 7, 2020. Yes, we do enjoy the role.
+* https://github.com/kubernetes/community/pull/4301
+ 
+What do you find challenging?
+* One of the challenges we have lies in cross SIG collaboration. It is challenging to get other SIG’s alignments on some of the features we are designing, especially when it requires a good amount of engagement and investment (both short term and long term wise). For example, we are working on the ContainerNotifier feature which needs support from SIG-Node. SIG-Node has assigned a reviewer for the KEP so that we can move forward.
+
+Do you have goals for the group?
+* Yes. As stated in the charter, the goal for this WG is to identify missing functionality and work together to design features to enable data protection support in Kubernetes.
+ 
+Do you want to continue or find a replacement? If you feel that you aren’t ready to pass the baton, what would you like to accomplish before you do?
+* We want to continue.
+ 
+Is there something we can provide that would better support you?
+* The most challenging thing is cross SIG collaboration. This WG is sponsored by both SIG-Storage and SIG-Apps. Recently we also need to collaborate with SIG-Node, and potentially with SIG-Arch. It will be great if the steering committee can provide guidance and help us get support from other SIGs.
+ 
+Do you have feedback for Steering? Suggestions for what we should work on?
+* If the Steering Committee can help us get support from other SIGs, that will be great.
+
+## Working Groups
+* What was the initial mission of the group and if it's changed, how?
+  * https://github.com/kubernetes/community/tree/master/wg-data-protection
+  * https://github.com/kubernetes/community/blob/master/wg-data-protection/charter.md
+  * The initial mission of this WG is to identify missing functionality and work together to design features to enable data protection support in Kubernetes. This is unchanged.
+ 
+* What’s the current roadmap until completion?
+  * We have many items listed in the in-scope section of the charter. We’d like to get them done.
```

Comment:
A few questions:
- Are there items in that list that are in progress currently? It might be a good idea to call it out here, especially for folks who aren't involved in the WG day-to-day but would like to learn more. :)
- Do the current projects have a destination they're tracking toward/when would be current projects be considered finished?

Reply:
Add this to the doc.